src/config/national.js

```diff
@@ -1,3 +1,4 @@
+// Configuration for the Ulaanbaatar section of the app
 export const NationalConfig = {
   metadata: {
```

Comment:
Not sure how to quickly describe this section. Any insight? I could just leave a note to not change it unless the schema of the Carto dataset defined in `metadata.table` changes.